### DISCUSSION TITLE
What should be the operator to use with a pgronga index if I want to to a string comparison ?

### DISCUSSION THREAD

**Discussion by ghevge** [pgroonga/pgroonga#390] [General]
In standard postgres we usually use like or ilike to do an exact string comparison, but with pgroonga, like and ilike are doing a full text search. 

So my question is, is there another operator that took the place of the original postgres like/ilike ?

Equal sign "=" cannot be used reliably with string comparisons as it won't always identify the differences, especially when special characters are involved.
eg:
'ரொட்டி' vs 'ரொட்டி'

These 2 strings are actually different, although they look identical. You can check that by calculating their hashes. But equal operator will return true when comparing them.

Another example, with pgroonga, like will return these 2 string as equal, even if at string level they are not:
"köttfärsbiff med 5% fett" vs "köttfärsbiff med 5 % fett"

Thanks

**Comment by askdkc** [comment]
@ghevge 

How about using `&=` operator?

https://pgroonga.github.io/reference/operators/exact-match-search.html
